bears/setup.in

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+$IMPORTS
```

Comment:
those are constant, why not have them in the template? The template is exactly for getting rid of the constants because they're not constants, tehy're template parts